### PR TITLE
🐛 Fix Supabase auth thread that prevents exit

### PR DESCRIPTION
**Pull request by fredericenard**
Here are related Supabase issues:
- https://github.com/supabase-community/supabase-py/issues/494
- https://github.com/supabase-community/supabase-py/issues/451

I'm not entirely sure removing sign out makes sense everywhere, fell free to put it back if needed.

### CODE REVIEW

**Comment by Koncopd** [comment]
Interesting, is it the reason our CI frequently hangs?

**Comment by Koncopd** [comment]
Reading the pinned threads, it looks to me that sign out should be left in the code and actually probably added at some other places.
Thank you very much for finding the root of the problem.

File: lamindb_setup/core/_hub_client.py [hunk 1]
```diff
@@ -62,6 +62,8 @@ def connect_hub(
     fallback_env: bool = False, client_options: ClientOptions = ClientOptions()
 ) -> Client:
     env = Environment(fallback=fallback_env)
+    client_options.auto_refresh_token = False
+    client_options.persist_session = False
```
Comment (falexwolf) [inline lamindb_setup/core/_hub_client.py:66]: Ah, I see!

Comment (Koncopd) [reply lamindb_setup/core/_hub_client.py:66]: Without sign out supabase tried to reuse the session between environments and it looks like anon tokens for different environments have different format, so the session tried to reuse sessions between environments and it resulted in a n error.

Comment (Koncopd) [reply lamindb_setup/core/_hub_client.py:66]: There is a problem that a session never expires so it sill hang forever in the auth table if we don't sign out and don't configure expiration time for **sessions** (not for access tokens).

**Pull request closed without merging**